bionic/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# SOURCE: https://github.com/William-Yeh/docker-ansible/blob/master/ubuntu18.04-onbuild/Dockerfile
+# There are no offical python ubuntu images available
+# And the PPA of Ansible still relies on Python 2
+FROM ubuntu:18.04
```

Comment:
`bionic` seems to be the code name of a specific ubuntu linux version.

I personally did not know the code name. and I always mix up the ones for Debian too. 

therefore I would prefer to use something like `ubuntu1804` as folder and later tag name.

(other opinions welcome..)